File: aplicação/frontend/ea-project/src/models/input/CreateStateInput.js
import BaseInput from "./BaseInput";

export default class CreateStateInput extends BaseInput {
  constructor(id = 0, name = '', creationDate = new Date(), lazyLoad = false) {
    super(lazyLoad);
    this.id = id;
    this.name = name;
    this.creationDate = creationDate;
  }
}

File: app/backend/src/main/java/eaproject/input/CreateStateInput.java
package eaproject.input;

import java.io.Serializable;
import java.sql.Timestamp;

public class CreateStateInput extends BaseInput implements Serializable {

    private String name;
    private Timestamp creationDate;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Timestamp getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(Timestamp creationDate) {
        this.creationDate = creationDate;
    }
}
